src/ApiPort/ApiPort.project.json

```diff
@@ -8,8 +8,8 @@
     "Microsoft.Framework.Configuration.CommandLine": "1.0.0-beta5",
     "Microsoft.Framework.OptionsModel": "1.0.0-beta5",
     "Newtonsoft.Json": "6.0.8",
-    "System.Collections.Immutable": "1.1.37",
-    "System.Reflection.Metadata": "1.1.0-alpha-00008",
+    "System.Collections.Immutable": "1.1.38-*",
+    "System.Reflection.Metadata": "1.2.0-alpha-00017",
```

Comment:
:) @conniey  was looking into updating this but there was a bug in the metadata reader she found that was fixed in this release.